### ISSUE TITLE
Elements with aria-hidden under element with role=tablist cause rule validation

### ISSUE THREAD

**Issue by Seaviello** [dequelabs/axe-core#4399]
### Product

axe-core

### Question

This question might be related to [issue](https://github.com/dequelabs/axe-core/issues/3938)

Currently following code 
```
<div role="tablist">
   <a aria-current="page" aria-selected="true" role="tab" href="...">Link 1</a>
   <div aria-hidden="true" ></div>
   <a aria-selected="false" role="tab" href="..." >Link 2</a>
</div>
```
cause a rule validation informing about child element being without `role="tab"`

When `aria-hidden` attribute is replaced with `role="presentation"` the validation error disappears. Using together `aria-hidden` with `role` attribute on the other hand still causes validation issue. Is that an expected behaviour?

**Comment by straker** [comment]
Thanks for the issue. Unfortunately we're not able to reproduce this issue. When I run the code I don't get any violations. Which rule is giving the violation (I assume `aria-required-child`)? 

Also which version of axe-core are you using? This sound similar to https://github.com/dequelabs/axe-core/issues/4090 which was fixed in [v4.8.0](https://github.com/dequelabs/axe-core/releases/tag/v4.8.0).

**Comment by Seaviello** [comment]
@straker We used version v4.6, updating solved an issue. Thank you and sorry for trouble.
